### PR TITLE
Use MariaDB account and secret in Designate controllers

### PR DESCRIPTION
**Pull request by omersch381**
So far, MariaDB account and secret were added to specs and types, and were used in designate_controller.go, but weren't used in the rest of the controllers.

This patch uses MariaDB in the rest of the controllers.

### CODE REVIEW

**Review by gthiemonge — CHANGES_REQUESTED** [review]

File: api/v1beta1/designate_types.go [hunk 1]
```diff
@@ -193,9 +193,6 @@ type DesignateStatus struct {
 	// Conditions
 	Conditions condition.Conditions `json:"conditions,omitempty" optional:"true"`
 
-	// Designate Database Hostname
-	DatabaseHostname string `json:"databaseHostname,omitempty"`
```
[the remark this answers is not in the document]
Comment (omersch381) [reply api/v1beta1/designate_types.go:197]: Done.

**Pull request merged**